poppy/tests/test_fresnel.py

```diff
@@ -789,44 +793,50 @@ def test_CompoundOpticalSystem_hybrid(npix=128):
                                err_msg="PSFs do not match between equivalent simple and compound/hybrid optical systems")
 
 
-def test_inwave_fresnel(plot=False):
```

Comment:
Something seems to have gone wrong here, possibly in one of the merge commits - I don't think you intend to delete the existing function `test_inwave_fresnel` here, do you? I assume your intent is to add a new additional function `test_FixedSamplingImagePlaneElement`, yes?   

We should not delete the test for the inwave functionality unless deleting that actual functionality, which I don't think is your intent here. 

[required fix, please]